gcp_variant_transforms/transforms/merge_conflicts.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Beam combiner function for merging VCF file header conflicts/definitions."""
+
+from collections import OrderedDict   #pylint: disable=unused-import
+
+import apache_beam as beam
+from gcp_variant_transforms.beam_io.vcf_header_io import VcfHeader
+from gcp_variant_transforms.libs.vcf_field_conflict_resolver import VcfParserConstants
+
+
+class _ConflictDetails(object):
+  """Container for conflict details for one field id."""
+  _MAX_FILE_NAME_NUM = 5
```

Comment:
Please add a comment about this constant as it's not obvious what it is for.

Reply:
Done.